### PR TITLE
[reactstrap] Don't declare "popper.js" module, prevent TypeScript conflict error

### PR DESCRIPTION
**Pull request by KurtPreston**
I recently submitted pull request #22512, which was merged.  However, upon updating my definitions, I realized that if a user has the `popper.js` node module installed, Typescript complains about conflicting type definitions.

This pull request removes the line re-declaring the popper.js module, and uses `import`s instead of `reference`s, keeping the definition internal.

I have copied this folder into my project's `node_modules/types/reactstrap` path, and it seems to work.  Please let me know if there is a better way to test it.

Also, apologies for pushing out an errant definition.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: **n/a — no external API changes, but rather fixing conflicting definition error**
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/reactstrap/lib/Popper.d.ts [hunk 1]
```diff
@@ -7,127 +7,130 @@
...
     scheduleUpdate(): void;
     enableEventListeners(): void;
     disableEventListeners(): void;
-  }
+}
 
-  declare module 'popper.js' {
-    export default Popper;
-  }
+export default Popper;
```
Comment (mhegazy) [inline types/reactstrap/lib/Popper.d.ts]: add `export as namespace Popper;` to make sure the module is still accessible as a global (which seems to be the case from the examples in https://github.com/FezVrasta/popper.js/).

**Review by mhegazy — CHANGES_REQUESTED** [review]
Please add `export as namespace Popper`.

**Comment by KurtPreston** [comment]
@mhegazy Updated.

**Comment by KurtPreston** [comment]
@mhegazy Thanks!

**Pull request merged**